src/operators/rasterscaling.md

```diff
@@ -0,0 +1,77 @@
+# RasterScaling
+
+The raster scaling operator scales/unscales the values of a raster by a given scale factor and offset.
+This allows to shrink and expand the value range of the pixel values needed to store a raster. It also allows to shift values to all-positive values and back. Keep in mind that scaling reduces the precision of the pixel values. We use the [GDAL](https://gdal.org/index.html) terms of [scale](https://gdal.org/programs/gdal_translate.html#cmdoption-gdal_translate-scale) and [unscale](https://gdal.org/programs/gdal_translate.html#cmdoption-gdal_translate-unscale). Scaling is often used to reduce memory consumption of rasters. Unscaling calculates the "real" value of each stored pixel.
+
+This is done by applying the following formulas to every pixel.
+
+For _unscaling_ the formula is: `p_new = p_old * slope + offset`.
+
+For _scaling_ the formula is: `p_new = (p_old - offset) / slope`
+
+`p_old` and `p_new` refer to the old and new pixel value. The slope and offset values are either properties attached to the input raster or a fixed value.
+
+An example for Meteosat Second Generation properties is:
+
+- offset: `msg.calibration_offset`
+- slope: `msg.calibration_slope`
+
+## Parameters
+
+| Parameter           | Type                                                  | Description                                          | Example Value |
+| ------------------- | ----------------------------------------------------- | ---------------------------------------------------- | ------------- |
+| `scaleWith`         | `MetadataKeyOrConstant`                               | the                                                  | "U8"          |
+| `offsetBy`          | `MetadataKeyOrConstant`                               | the output type                                      | "U8"          |
```

Comment:
description and example are wrong

Reply:
i reworked the docs in this commit: [aaa9676](https://github.com/geo-engine/docs/pull/33/commits/aaa967690250857a15aecf0b2993d2172004f14f)